### PR TITLE
Update post from 2020 on using Deno

### PR DESCRIPTION
**Pull request by alisaduncan**
Updates [Build your First Deno App with Authentication](https://developer.okta.com/blog/2020/09/14/deno-with-auth) to use newer dependencies.

Example app updates in [oktadev/okta-deno-auth-example/pull/3](https://github.com/oktadev/okta-deno-auth-example/pull/3).

### CODE REVIEW

**Review by mraible — APPROVED** [review]
Looks great! 👍

File: _source/_posts/2020-09-14-deno-with-auth.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ tweets:
 - "Want to learn the basics of Deno and build your first Deno app? Check this out!"
 image: "blog/first-deno-with-auth/social-image.png"
 type: conversion
+changelog: 
+  - 2021-12-010: Updated GitHub repo and blog post with dependency updates. You can see the changes in the [example app on GitHub](https://github.com/oktadev/okta-deno-auth-example/pull/3). Changes to this article can be viewed in [oktadev/okta-blog#991](https://github.com/oktadev/okta-blog/pull/991).
+---
 ---
 
 The creator of Node.js, Ryan Dahl has authored a new framework for designing web applications. He went back and fixed some mistakes he made in hindsight, taking advantage of new technologies that were not available at the time he originally wrote Node. The result is [Deno](https://deno.land) (pronounced DEH-no), a framework for writing "Node-like" web applications in TypeScript. Here, I will walk you through creating a basic web application with authentication.
```
Comment (havi) [inline _source/_posts/2020-09-14-deno-with-auth.md]:
```suggestion
The creator of Node.js, Ryan Dahl, has authored a new framework for designing web applications. He went back and fixed some mistakes he made in hindsight, taking advantage of new technologies that were not available at the time he originally wrote Node. The result is [Deno](https://deno.land) (pronounced DEH-no), a framework for writing "Node-like" web applications in TypeScript. Here, I will walk you through creating a basic web application with authentication.
```
comma

File: _source/_posts/2020-09-14-deno-with-auth.md [hunk 2]
```diff
@@ -158,15 +161,44 @@ This includes the footer and header partials and adds a link to the profile page
 
 Again, this page includes the header and footer, and loops through the properties of the `user` object. Granted, it's not a super-sexy profile page, but it will let you know that the authentication steps all worked.
 
-## Add Authentication with Okta
 
-If you don't already have an Okta account, you can get a free developer account [here](https://developer.okta.com/signup). Once you've signed into Okta, you'll land on the dashboard. You'll need to create an Okta application to take advantage of Okta as an Identity Provider for your project.
+## Manage your dependencies
+
+Now that you created your second file and imported the same dependencies, it's easy to see what a chore managing and updating them will be. Deno recommends placing all imports in a single `deps.ts` file to centralize dependencies.
```
Comment (havi) [inline _source/_posts/2020-09-14-deno-with-auth.md]:
```suggestion
Now that you've created your second file and imported the same dependencies, it's easy to see what a chore managing and updating them will be. Deno recommends placing all imports in a single `deps.ts` file to centralize dependencies. 
```
past perfect tense

File: _source/_posts/2020-09-14-deno-with-auth.md [hunk 3]
```diff
@@ -158,15 +161,44 @@ This includes the footer and header partials and adds a link to the profile page
 
 Again, this page includes the header and footer, and loops through the properties of the `user` object. Granted, it's not a super-sexy profile page, but it will let you know that the authentication steps all worked.
 
-## Add Authentication with Okta
 
-If you don't already have an Okta account, you can get a free developer account [here](https://developer.okta.com/signup). Once you've signed into Okta, you'll land on the dashboard. You'll need to create an Okta application to take advantage of Okta as an Identity Provider for your project.
+## Manage your dependencies
```
Comment (havi) [inline _source/_posts/2020-09-14-deno-with-auth.md]:
```suggestion
## Manage Your Dependencies
```
consistent use of title case in headings to match w/rest of post

**Review by havi — COMMENTED** [review]
3 nitty edits batched together. Feel free to commit the batch and publish.

**Pull request merged**